cmd/walletd/main.go

```diff
@@ -204,6 +204,8 @@ func main() {
 	rootCmd.BoolVar(&cfg.Log.File.Enabled, "log.file.enabled", cfg.Log.File.Enabled, "enable file logging")
 	rootCmd.BoolVar(&cfg.Log.StdOut.Enabled, "log.stdout.enabled", cfg.Log.StdOut.Enabled, "enable stdout logging")
 
+	rootCmd.TextVar(&cfg.Checkpoint, "checkpoint", cfg.Checkpoint, "instant-sync to a chain index")
```

Comment:
It would be nice to have the description contain an example value. To make it a bit more obvious how to use this.